### PR TITLE
feat: add limit bar and analytics

### PR DESCRIPTION
**Pull request by davidsoderberg**
Screenshot 2023-01-18 at 09 34 17
Screenshot 2023-01-18 at 09 34 24

### CODE REVIEW

File: apps/web/src/components/layout/components/HeaderNav.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { Tooltip } from '../../../design-system';
 import { INTERCOM_APP_ID } from '../../../config';
 import { SpotlightContext } from '../../../store/spotlightContext';
 import { HEADER_HEIGHT } from '../constants';
+import { LimitBar } from '../../../pages/integrations/components/LImitBar';
```
Comment (p-fernandez) [inline apps/web/src/components/layout/components/HeaderNav.tsx:16]:
```suggestion
import { LimitBar } from '../../../pages/integrations/components/LimitBar';
```
Filename spelling. 🙂

File: apps/web/src/components/templates/BlueprintModal.tsx [hunk 2]
```diff
@@ -8,17 +8,34 @@ import { When } from '../utils/When';
...
 
+  const { mutateAsync: updateOnBoardingStatus } = useMutation<
+    IUserEntity,
+    { error: string; message: string; statusCode: number },
+    { showOnBoarding: boolean }
+  >(({ showOnBoarding }) => updateUserOnBoarding(showOnBoarding));
+
+  async function disableOnboarding() {
+    await updateOnBoardingStatus({ showOnBoarding: false });
+  }
```
Comment (p-fernandez) [inline apps/web/src/components/templates/BlueprintModal.tsx:37]: We should be coherent with the spelling to not be confusing.
Checking the right spelling in the dictionary I think we should change to `updateUserOnboarding`, `updateOnboardingStatus` and `showOnboarding`.

**Review by p-fernandez — APPROVED** [review]
Very helpful the screenshots to understand what's going on for this feature.

File: apps/web/src/components/templates/BlueprintModal.tsx (hunk 2 above)
Comment (davidsoderberg) [reply apps/web/src/components/templates/BlueprintModal.tsx:37]: .... copied it from another part of our code and I think the object property needs to be `showOnBoarding`

Comment (p-fernandez) [reply apps/web/src/components/templates/BlueprintModal.tsx:37]: Ouch, I have just seen that is a schema property also. That one we shouldn't touch it. Damn. 😞

File: apps/web/src/components/templates/BlueprintModal.tsx [hunk 3]
```diff
@@ -39,6 +56,7 @@ export function BlueprintModal() {
     onSuccess: (template) => {
       localStorage.removeItem('blueprintId');
       if (template) {
+        disableOnboarding();
```
Comment (LetItRock) [inline apps/web/src/components/templates/BlueprintModal.tsx:59]: Why we are disabling the onboarding? I feel like for the new accounts they won't still know how to do integrations, create the template, use trigger etc.

**Pull request merged**